app/views/amend/ViewOrChangePreviousRegistrationsMultipleView.scala.html

```diff
@@ -0,0 +1,51 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import controllers.amend.routes
+@import models.amend.PreviousRegistration
+@import models.amend.PreviousRegistration.options
+
+@this(
+        layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        govukErrorSummary: GovukErrorSummary,
+        govukRadios: GovukRadios,
+        govukButton: GovukButton
+)
+
+@(form: Form[_], waypoints: Waypoints, previousRegistrations: Seq[PreviousRegistration])(implicit request: Request[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("viewOrChangePreviousRegistrationsMultiple.title"))) {
+
+    @formHelper(action = routes.ViewOrChangePreviousRegistrationsMultipleController.onSubmit(waypoints), Symbol("autoComplete") -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value_0")))
```

Comment:
double checking, is `value_0` expected here?

Reply:
Just auto generated and don't need index for anything specific as error message is generic.